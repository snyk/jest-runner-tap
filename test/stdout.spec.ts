import { testTranslation } from './sniff';

describe('maintenance of stdout', () => {
  it('handles a simple console.log', async () => {
    const result = await testTranslation('./fixtures/stdout.tap');
    expect(result.numPassingTests).toBe(1);
    expect(result.failureMessage).toBeUndefined();
    expect(result.console).toMatchSnapshot();
  });
});
