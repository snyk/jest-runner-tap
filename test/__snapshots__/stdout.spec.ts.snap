// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`maintenance of stdout handles a simple console.log 1`] = `
Array [
  Object {
    "message": "hello there",
    "origin": "tap",
    "type": "log",
  },
]
`;
